mesonbuild/mtest.py

```diff
@@ -2143,7 +2150,7 @@ def convert_path_to_target(path: str) -> str:
     return True
 
 def run(options: argparse.Namespace) -> int:
-    if options.benchmark:
+    if options.benchmark or options.interactive:
         options.num_processes = 1
```

Comment:
I feel like it would be good to mention this in the documentation of the feature.